hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -203,7 +203,30 @@
     public static final String PROP_ELASTIC_MEMORY_UNSAFE_ENABLED = "hazelcast.elastic.memory.unsafe.enabled";
     public static final String PROP_ENTERPRISE_LICENSE_KEY = "hazelcast.enterprise.license.key";
     public static final String PROP_MAP_WRITE_BEHIND_QUEUE_CAPACITY = "hazelcast.map.write.behind.queue.capacity";
+
+    /**
+     * Defines event queue size for WAN replication, only valid for Hazelcast Enterprise
+     */
     public static final String PROP_ENTERPRISE_WAN_REP_QUEUESIZE = "hazelcast.enterprise.wanrep.queuesize";
+
+    /**
+     * Defines maximum number of WAN replication events to be gathered in a batch, only valid for Hazelcast Enterprise
+     */
+    public static final String PROP_ENTERPRISE_WAN_REP_BATCHSIZE = "hazelcast.enterprise.wanrep.batchsize";
+
+    /**
+     * Defines batch sending frequency in seconds,
+     * Used in case queued event count does not reach to {@link this#PROP_ENTERPRISE_WAN_REP_BATCHSIZE}
+     * Only valid for Hazelcast Enterprise
+     */
+    public static final String PROP_ENTERPRISE_WAN_REP_BATCH_FREQUENCY = "hazelcast.enterprise.wanrep.batchfrequency";
+
+    /**
+     * Defines timeout duration (in milliseconds) for a WAN replication event to reach target cluster,
+     * only valid for Hazelcast Enterprise
+     */
+    public static final String PROP_ENTERPRISE_WAN_REP_OP_TIMEOUT = "hazelcast.enterprise.wanrep.optimeout";
```

Comment:
Please add the unit, e.g. PROP_EVENT_QUEUE_TIMEOUT_MILLIS. Also include it in the string: hazelcast.enterprise.wanrep.optimeout.millis.